src/tests/DetailsPage.test.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable max-len */
```

Comment:
- [ ] kindly ensure that no linter is disabled, linters are tools that analyze your code to identify and flag potential issues, errors, or stylistic inconsistencies. They are widely used in software development to ensure code quality and maintainability. so kindly try to solve the issue not disable it 💯💪🏻⭐ I am sure you will find a solution for it 👏🏻💪🏻💯